webapp/src/AppComponents/FileBar.tsx

```diff
@@ -107,7 +125,7 @@ const Toast = styled("div")`
 `;
 
 const Divider = styled("div")`
-  margin: 10px;
+  margin: 0px 8px 0px 16px;
```

Comment:
Left over from another PR?

Reply:
Hah no. Even if it's looks the opposite, it's a way to make logo, divider and File button look like they are at the same distance...